api-samples/devtools/panels/chrome-query/devtools.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2012 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// The function below is executed in the context of the inspected page.
+/*global $0*/
+let page_getProperties = function () {
+  let data = window.jQuery && $0 ? jQuery.data($0) : {};
```

Comment:
We should probably do a pass updating the code style where we can (e.g, we could use const here).